core/src/models/BibleInput.ts

```diff
@@ -17,12 +17,13 @@ export interface IBibleContentSectionForInput extends IBibleSectionBase {
     contents: IBibleContentForInput[];
 }
 
-export interface IBibleContentGroupForInput<T extends IContentGroup['groupType']>
-    extends IContentGroup {
+export interface IBibleContentGroupForInput<
+    T extends IContentGroup['groupType']
+> extends IContentGroup {
     readonly type: 'group';
     readonly groupType: T;
 
-    contents: (IBibleContentGroupForInput<T> | IBibleContentPhraseForInput)[];
+    contents: (IBibleContentGroupForInput<any> | IBibleContentPhraseForInput)[];
```

Comment:
i think we should (almost) never use `any`. in this case i think what we need is `IContentGroup['groupType']`